sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -71,7 +71,9 @@ object CurrentOrigin {
   }
 }
 
+// scalastyle:off
```

Comment:
would be better to turn off style check for this specific rule so it is more clear since it is not obvious why this needs to be excluded looking at this line.